validation/standard/init_test.go

```diff
@@ -28,18 +28,28 @@ func TestConfigInitialization(t *testing.T) {
 	require.NotNil(t, Config.MultisigRoles, "Config.MultisigRoles should not be nil")
 
 	// Check individual network configurations
-	networks := []string{"mainnet", "sepolia"}
-	for _, network := range networks {
+	for network, params := range Config.Params {
 		t.Run(fmt.Sprintf("Params[%s]", network), func(t *testing.T) {
 			// Ensure network Params are populated
-			require.NotNil(t, Config.Params[network], "Config.Params[%s] should not be nil", network)
-			require.NoError(t, Config.Params[network].Check(), "Config.Params[%s] has invalid zero value", network)
+			require.NotNil(t, params, "Config.Params[%s] should not be nil", network)
+			require.NoError(t, params.Check(), "Config.Params[%s] has invalid zero value", network)
 		})
 
 		t.Run(fmt.Sprintf("MultisigRoles[%s]", network), func(t *testing.T) {
+			roles := Config.MultisigRoles[network]
+
 			// Ensure network MultisigRoles are populated
-			require.NotNil(t, Config.MultisigRoles[network], "Config.MultisigRoles[%s] should not be nil", network)
-			require.NotZero(t, Config.MultisigRoles[network], "Config.MultisigRoles[%s] should not be zero value", network)
+			require.NotNil(t, roles, "Config.MultisigRoles[%s] should not be nil", network)
+			if roles != nil {
```

Comment:
Nit: since you require that `roles != nil` on the line above, don't think you need this `if` statement. Same for the `if l1Roles != nil` statement below

Reply:
my thinking was that since `require.NotNil` only fails the test, the program would continue and could potentially panic on the next line, and that could confuse the test output

Reply:
I think `require.NotNil` will exit the test execution immediately upon failure. This is different from `assert.NotNil`, which behaves the way you describe

Reply:
nvm tested again and you're right, i'll make this change